### ISSUE TITLE
Build failure with fsnotify 4.0

### ISSUE THREAD

**Issue by bwbeach** [phoityne/haskell-debug-adapter#26]
I set up a new Haskell environment, and tried installing `haskell-debug-adapter`, but got an error about missing exports from fsnotify.  After cloning this repo and trying to build, I get:

```
src/Haskell/Debug/Adapter/Watch.hs:38:32: error:
    Not in scope: ‘S.confDebounce’
    Module ‘System.FSNotify’ does not export ‘confDebounce’.
   |
38 |   let conf = S.defaultConfig { S.confDebounce  = S.Debounce 1} 
   |                                ^^^^^^^^^^^^^^

src/Haskell/Debug/Adapter/Watch.hs:38:50: error:
    Not in scope: data constructor ‘S.Debounce’
    Module ‘System.FSNotify’ does not export ‘Debounce’.
   |
38 |   let conf = S.defaultConfig { S.confDebounce  = S.Debounce 1} 
   |                                                  ^^^^^^^^^^
```

The [changelog for fsnotify](https://github.com/haskell-fswatch/hfsnotify/blob/master/CHANGELOG.md) says that debouncing was pulled out of the library in version 4.0, and the major version was bumped because it's a breaking change.  This happened on 2022-09-06.

**Comment by bwbeach** [comment]
I've tried simply getting rid of the debounce option.  Tests pass, and I'm able to set breakpoints while debugging.  Happy to create a PR if this is a reasonable solution.

```
+++ b/src/Haskell/Debug/Adapter/Watch.hs
@@ -35,7 +35,7 @@ app = flip catchError errHdl $ do
   liftIO $ L.infoM _LOG_WATCH $ "start watching " ++ ws
 
   reqStore <- view reqStoreAppStores <$> get
-  let conf = S.defaultConfig { S.confDebounce  = S.Debounce 1} 
+  let conf = S.defaultConfig
   liftIO $ S.withManagerConf conf $ goIO ws reqStore
```

**Comment by phoityne** [comment]
Thanks for the report.
I also realized it and have already fixed in local repository.

I'm going to check in and release.
Regards.

**Comment by bwbeach** [comment]
Thanks!

**Comment by hanjoosten** [comment]
This also broke my devcontainer for haskell. Any idea when this will be released?

**Comment by phoityne** [comment]
released hda-0.0.36.0.
fixed.

**Comment by hanjoosten** [comment]
@phoityne , was there a need not to support ghc-8 any longer? I have a project where not all dependencies are available for ghc-9.

**Comment by phoityne** [comment]
Released hda-0.0.37.0.
reverted for supporting the ghc8.
